Pipfile

```diff
@@ -19,7 +24,7 @@ requests = "*"
 fsspec = "==0.8.4"
 aiohttp = "==3.7.4"
 pyyaml = "==5.4.1"
-JSON-log-formatter = "==0.5.0"
+fybrik_python_logging = {version="0.6.2", index="testpypi"}
```

Comment:
You might want to use version `0.0.0` in master version of arrow-flight-module. the release version of the library should be used only when afm is released. You can use a script similar to what is done in https://github.com/fybrik/fybrik/blob/master/.github/workflows/release-branch.yaml